docs/plugins/mapper-attachments.asciidoc

```diff
@@ -1,6 +1,9 @@
 [[mapper-attachments]]
 === Mapper Attachments Plugin
 
+deprecated[3.0.0, Mapper attachments has been deprecated by <<ingest-attachment>>]
+
```

Comment:
Gah, sorry.  Just realised this is in plugins, so:

    deprecated[3.0.0,The `mapper-attachments` plugin has been replaced by the <<ingest-attachment, `ingest-attachment`>> plugin]